README.md

```diff
@@ -1,45 +1,191 @@
+            },
+            // SetFlags are chart value overrides
+            SetFlags: types.Flags{
+                ".some.value.override": "override",
+            },      
+        },
+    },
+    ClusterInfo: custom.ClusterInfo{
+        Config: restConfig, // destination cluster rest config
+        Client: client, // destination cluster rest client
+    },
+    ResourceInfo: manifest.ResourceInfo{
+        CustomResources: []*unstructured.Unstructured{} // additional custom resources to be installed
+        BaseResource: unstructured.Unstructured{}, // base resource to be passed for custom checks, usually the reconciled resource
+		Crds: []*apiextensions.CustomResourceDefinition // additional custom resource definitions to be installed
+    },
+    CheckReadyStates: true,
+}
+
+ready, err := manifest.InstallChart(logger, deployInfo, []types.ObjectTransform{})
+if err != nil {
+	return false, err
+}
+
+ready, err := manifest.UninstallChart(logger, deployInfo, []types.ObjectTransform{})
+if err != nil {
+    return false, err
+}
+
+ready, err := manifest.ConsistencyCheck(logger, deployInfo, []types.ObjectTransform{})
+if err != nil {
+    return false, err
+}
+
 ```
-  args = map[string]string{
-    "set": "xyz=abc",
-    "flags": "CreateNamespace=true,Namespace=new-space",
-  }
-  ```
+
+### Local setup
+
+- Refer to the local cluster and registry setup [documentation](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/developer/provision-cluster-and-registry.md#local-cluster-setup) to create cluster(s) either in a single-cluster or dual-cluster mode.
+- Set your `KUBECONFIG` environment variable to point towards the desired cluster.
+- Run the following `make` file commands:
+
+| Make command | Description                                      |
+|--------------|--------------------------------------------------|
+| build        | Run fmt, vet and DeepCopy method implementations |
+| manifests    | Create CRDs based on [API types](operator/api)   |
+| install      | Install CRDs                                     |
+| run          | Run operator controller locally                  |
+
+### Cluster setup
+
+- Refer to the remote cluster and registry setup [documentation](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/developer/provision-cluster-and-registry.md#remote-cluster-setup) to create cluster(s) either in a single-cluster or dual-cluster mode.
+- Set your `KUBECONFIG` environment variable to point towards the desired cluster.
+- Run the following commands with `IMG` environment variable pointing to the image name
+
+| Make command | Description                                           |
+|--------------|-------------------------------------------------------|
+| docker-build | Build operator image                                  |
+| docker-push  | Push docker image to your repo                        |
+| deploy       | Deploys the operator resources to the desired cluster |
+
+### Contribution
```

Comment:
We have a Kyma Contribution Guideline that we should reference here I think: https://kyma-project.io/community/contributing/02-contributing/